src/openms/include/OpenMS/CHEMISTRY/ElementDB.h

```diff
@@ -113,6 +124,30 @@ namespace OpenMS
                     const std::map<unsigned int, double>& abundance,
                     const std::map<unsigned int, double>& mass,
                     bool replace_existing);
+
+    /** Adds or replaces a new isotope in the database
+     *
+     * Adds a new isotope (or replaces an existing one if @em replace_existing is true). 
+     *
+     * @param name Common name of the element
+     * @param symbol Element symbol (one or two letter)
+     * @param an Atomic number (number of protons)
+     * @param abundance natural abundance of the isotope (use 0 for unstable isotopes)
+     * @param mass mass of the isotope
+     * @param half_life Half life of the isotope in seconds (use -1 for stable isotopes)
+     * @param decay Main decay mode for unstable isotopes (use NONE for stable isotopes)
+     * @param replace_existing Whether to replace an existint isotope
```

Comment:
```suggestion
     * @param replace_existing Whether to replace an existing isotope
```
maybe document what happens if replace_existing = false and an existing one is passed (no-op? exeption?)